src/annotate/random_forest/config.vsh.yaml

```diff
@@ -0,0 +1,125 @@
+functionality:
+  name: random_forest
```

Comment:
Could you propose a more specific name for the component? Random forrest is used in many applications and I would like to be a little bit more specific here to avoid conflicting component names